### PR TITLE
README update

### PR DESCRIPTION
**Pull request by sophoah**
Fix formatting for better view of the README
Add notice for bitcoin transfer signature
Add what is a bitcoin outpoint

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -209,15 +219,18 @@ To send wrapped Bitcoin back to Bitcoin, run the following commands:
     ```
     axelarcli tx bitcoin sign-pending-transfers 0.0001btc --gas="auto" --gas-adjustment=1.15 --from validator -b block -y
     ```
-5. Submit the transfer to Bitcoin
+
+    **Note**: If you are having gas error issue `out of gas: out of gas in location: WriteFlat; gasWanted: 200000, gasUsed: 200780"`, try to increment the `gas-adjustement` to a higher value `1.2` or `1.3`. With more deposits, the transaction become larger and hence requires more gas
```
Comment (sergeynog) [inline README.md:223]: i would just change the command to add auto-gas flags 
`— gas="auto" — gas-adjustment=1.15`

File: README.md [hunk 2]
```diff
@@ -209,15 +219,18 @@ To send wrapped Bitcoin back to Bitcoin, run the following commands:
...
+3. Submit the transfer to Bitcoin
     ```
     axelarcli q bitcoin send
     -> returns tx ID
     ```
     You can monitor the status of your transfer using the bitcoin testnet explorer: https://blockstream.info/testnet/ .
 
 #### 🛑 **IMPORTANT: Verify outpoints of previous withdrawal tx (repeat for each outpoint)**
-Without this step, other users of the testnet will be unable to withdraw their wrapped tokens. Be a good citizen and verify the outpoints!
+    Without this step, other users of the testnet will be unable to withdraw their wrapped tokens. Be a good citizen and verify the outpoints!
```
Comment (sergeynog) [inline README.md:233]: should we clarify what an `outpoint` is? i saw some folks were confused.

**Review by sergeynog — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mindstyle85) [reply README.md:233]: yes i think its best to clarify

File: README.md (hunk 1 above)
Comment (sophoah) [reply README.md:223]: `--gas=auto` flag is already present, would that flag alone suffice ? I had in the past to also add the flag --gas-adjustement to get a tx signature to work

File: README.md (hunk 2 above)
Comment (sophoah) [reply README.md:233]: @sergeynog yes agree, if you provide the definition here, i can update the PR, I am also not clear here, just guesses based on the verification that was required.

Comment (sophoah) [reply README.md:235]: @sergeynog I added a definition of a bitcoin outpoint

File: README.md (hunk 1 above)
Comment (sergeynog) [reply README.md:223]: ah, i didn't see that we already updated the command. with auto flags.

File: README.md (hunk 2 above)
Comment (sergeynog) [reply README.md:235]: how about this: replace "Bitcoin identifies transactions using a hash and an outpoint is a hash and index pair in which the index simply identifies which output a given transaction is intended." -> "Every Bitcoin transaction can have multiple outputs. Similarly, on your withdrawal transactions you will see multiple outputs even if you processed one withdrawal (the other outputs are used for internal controller functions). Please verify all outputs (i.e., {transaction hash, index} pairs) in your withdrawal transaction.

**Review by sergeynog — APPROVED** [review]

**Pull request merged**